### PR TITLE
add migrations and phone verification

### PR DESCRIPTION
**Pull request by JonasWijne**

### CODE REVIEW

File: kyc/migrations.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+migrations_list = [
+    ("0", "CREATE TABLE IF NOT EXISTS users (user_id text NOT NULL UNIQUE,email text NOT NULL,verification_code text NOT NULL, verified integer,public_key text NOT NULL,signed_email_identifier text NULL);"),
+    ("0", "CREATE TABLE IF NOT EXISTS migrations (migration_id text NOT NULL);"),
```
Comment (tobiaschielens) [inline kyc/migrations.py:3]: 2x id 0

File: kyc/routes.py [hunk 2]
```diff
@@ -77,20 +83,106 @@ def verify_handler():
...
+
+    union = "?"
+
+    if union in redirect_url:
+        union = "&"
+
+    url = "{}{}userId={}&verificationCode={}".format(redirect_url, union, user_id, sms_verification_code)
+    logger.debug("url: %s", url)
+
+    body = "Hi {} <br><br> You have just created a ThreeFold Connect account.<br> On behalf of the ThreeFold Connect team, we hereby provide a link to verify your email address. When you click on this link, you will be taken to a page confirming your address is verified.<br> Without this verification, not all features will be available.<br> <a href=""{}>Verify my email address</a><br><br> Thanks,<br>OpenKYC Team".format(
```
Comment (tobiaschielens) [inline kyc/routes.py:130]: sms body contains email instead of sms

File: kyc/migrations.py (hunk 1 above)
Comment (JonasWijne) [reply kyc/migrations.py:3]: jup, its an example, this doesnt need to happen and will be properly skipped

File: kyc/routes.py (hunk 2 above)
Comment (JonasWijne) [reply kyc/routes.py:130]: copy needs to change indeed

Comment (JonasWijne) [reply kyc/routes.py]: copy updated:

```
Hello {},

If you requested a phone verification for ThreeFold Connect, please click the following link.

{}

Thanks,
OpenKYC Team
```

**Pull request merged**